package.json

```
@@ -62,7 +62,7 @@
     "@types/classnames": "^2.2.11",
     "@types/gapi": "^0.0.39",
     "@types/prop-types": "^15.7.3",
-    "@types/query-string": "^5.0.0",
+    "@types/query-string": "^6.3.0",
```

```suggestion
    "@types/query-string": "^6.3.0",
```
@dependabot ignore this major version